camayoc/tests/qpc/api/v1/reports/test_reports.py

```diff
@@ -334,3 +335,62 @@ def test_raw_facts_details_report(scans, scan_name):
             scan_results=pformat(report_content),
         )
     )
+
+
+@pytest.mark.runs_scan
+@pytest.mark.parametrize("scan_name", scan_names(has_aggregate))
+def test_aggregate_report(scans, scan_name):
+    """Test that aggregate values are correct for the source.
+
+    :id: dd310642-75b3-47ca-a410-523f9f541ceb
+    :description: Test that aggregate values are correctly tallied for the source.
+    :steps:
+        1) Request the aggregate json report for the scan.
+        2) Assert that the found aggregate values match these listed
+           in the configuration file for the source.
```

Comment:
```suggestion
           in the configuration file for the scan.
```

Expected aggregate results in configuration file are specified for a scan, not a source.

There are more references to "source" in the docstring here.